### PR TITLE
smart-qa 0.0.1

### PR DESCRIPTION
**Pull request by kot10n5tn**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/smart-qa/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+title: SmartQA
+summary: |
+  SmartQA App.
```
Comment (koral--) [inline steps/smart-qa/0.0.1/step.yml:3]: This does not tell anything.

File: steps/smart-qa/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+project_type_tags:
+- ios
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/Levetty/bitrise-step-smartqa
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/smart-qa/0.0.1/step.yml:25]: Those deps should be removed. You are using go toolkit.

File: steps/smart-qa/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_path: null
+  opts:
+    description: |
+      Description of this input.
```
Comment (koral--) [inline steps/smart-qa/0.0.1/step.yml:38]: not changed template

File: steps/smart-qa/0.0.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      Can be Markdown formatted text.
+    is_expand: true
+    is_required: true
+    summary: Your application build path.
+    title: Build Path
+- api_key: null
+  opts:
+    description: |
+      Description of this input.
+      Can be Markdown formatted text.
```
Comment (koral--) [inline steps/smart-qa/0.0.1/step.yml:48]: Why anyone would have to format api key?

File: steps/smart-qa/0.0.1/step.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+inputs:
+- build_path: null
+  opts:
+    description: |
+      Description of this input.
+      Can be Markdown formatted text.
+    is_expand: true
+    is_required: true
+    summary: Your application build path.
+    title: Build Path
```
Comment (koral--) [inline steps/smart-qa/0.0.1/step.yml:43]: What does build path mean? An example should be described.

**Comment by kot10n5tn** [comment]
Thanks.
I have fixed the lines. How can I update my PR?
Do I need to use `bitrise stepman` command?

**Comment by koral--** [comment]
The fastest way is to just repeat the share process. Remember to include only one commit.

**Comment by kot10n5tn** [comment]
Thank you. Please review.

**Review by trapacska — APPROVED** [review]

**Pull request merged**